savaslabs/savaslabs.github.io
# Issue #3877: Update url to use https

This updates the site URL in Jekyll config to use `https`, which we'll need when we enable SSL for the site and redirect `http` to `https`. This is necessary for URLs to images generated by the jekyll-picture-tag plugin and links to assets like JS.

## To test

1. Pull this branch and run `gulp clean` then `gulp build`. This will build the `_site` directory as it's built for production.
2. Search the entire codebase for http://savaslabs.com. You shouldn't find anything.
3. Open up the file `_site/company/index.html`. Search for `region--team-member-list`; around line 117. Check out the URLs in the `sourceset` and `src` attributes in the picture tags for team member photos - the links should use `https`.
4. View a blog post file, such as `_site/2017/04/24/savas-labs-website-redesign.html`. Scroll all the way to the end of the file, around line 1462, to find the `<script>` tag loading our main JS file. The src URL should be https://savaslabs.com/assets/js/main.js.
5. You can also check out some of the schema in the blog post file or company page file to ensure the URLs to our site use `https`. To do this, search the file for `"@context": "http://schema.org"` to find the schema JSON. We'll also want to run the site through Google's structured data checker after we deploy this.

Let me know if you find any issues or have any questions! Once approved, I will merge (need to coordinate merge with updates in CloudFlare).
